.fwd

```diff
@@ -29,6 +29,8 @@ FWD_DOCKER_BIN=docker
 FWD_DOCKER_COMPOSE_BIN=docker-compose
 # Docker Compose Exec flags
 FWD_COMPOSE_EXEC_FLAGS=""
+# Docker Compose Run flags
+FWD_COMPOSE_RUN_FLAGS="--rm"
```

Comment:
i think it may be good to default it to `-it --init` too

Reply:
also to follow docker run, `--rm` is always injected, so maybe add it to builder directly and on the custom flags just `-it --init`

https://github.com/fireworkweb/fwd/blob/ab88db2cb43e28ccf37d63fa9f016d4f445c5d43/app/Builder/DockerRun.php#L20

Reply:
`docker-compose run` doesn't have `-it` and `--init` flags:

https://docs.docker.com/compose/reference/run/